test/service/EvidenceUploadService.test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from 'chai';
+import * as FormData from 'form-data';
+import nock = require('nock');
+
+import { EvidenceUploadService } from 'app/service/EvidenceUploadService'
+
+describe('EvidenceUploadService', () => {
+
+    const KEY: string = 'mockKey';
+    const HOST: string = 'http://localhost';
+    const evidenceUploadService = new EvidenceUploadService(HOST , KEY);
+
+    describe('upload file', () => {
```

Comment:
According to API definition it can also return 415. Looks like we need to handle, only question is when. If we need to handle such cases where MIME was not supported in a nicer way than rendering unexpected error page we should add this to case to the test.

https://github.com/companieshouse/file-transfer-api/blob/master/specs/schema.json#L56

Reply:
I double checked and it looks like MIME validation is within the scope of the story. It means that 415 must be handled properly and relevant test must be in place before merge.